vue/components/ui/molecules/select/select.vue

```diff
@@ -158,6 +159,10 @@ export const Select = {
             type: Boolean,
             default: false
         },
+        width: {

```

Comment:
Dead code?

Reply:
This prop was probably removed as there is code that uses it. See https://github.com/ripe-tech/ripe-components-vue/blob/a1ce5d7296d0d134c71c5ad3104bd5c249b439e5/vue/components/ui/molecules/select/select.vue#L383
Also the prop width is being used here for example: https://github.com/ripe-tech/ripe-components-vue/blob/a1ce5d7296d0d134c71c5ad3104bd5c249b439e5/vue/components/ui/molecules/search-persistent/save-filter-modal.vue#L32